pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -333,6 +333,7 @@ type EndpointStatus struct {
 const EndpointStatusLogEntries = 5
 
 // ControllerList is a list of ControllerStatus
+// +k8s:deepcopy-gen=false
```

Comment:
What makes these structs "unnecessary"? Update the commit to provide more context regarding this.

Reply:
Done